test/webgl-utils/parameters.spec.js

```diff
@@ -0,0 +1,168 @@
+import {createTestContext} from '../setup';
+import {GL} from 'luma.gl';
+import {glSetParameters, glGetParameter, glCopyParameters, GL_PARAMETER_DEFAULTS}
+  from '../../src/webgl-utils/parameters';
+import trackContext from '../../src/webgl-utils/track-context';
+import test from 'tape-catch';
+
+// Settings test, don't reuse a context
+const fixture = {
+  gl: createTestContext({debug: true})
+};
+
+// eslint-disable-next-line
+const GL_PARAMETER_SETTINGS_ONE = {
```

Comment:
We could copy these into a data file since we are using them in multiple test files.